theano/tensor/tests/test_basic.py

```diff
@@ -5188,6 +5189,37 @@ def test_infer_shape(self):
         assert numpy.all(f(0) == len(numpy.arange(0, 0)))
 
 
+class TestNdGrid(unittest.TestCase):
```

Comment:
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "theano/tensor/basic.py", line 4399, in __getitem__
    ndim = len(args[0])
TypeError: object of type 'slice' has no len()

This case fail, so it need a test.